packages/opencensus-web-core/src/trace/model/tracer-base.ts

```diff
@@ -76,6 +80,13 @@ export class TracerBase implements webTypes.TracerBase {
    */
   startRootSpan<T>(options: webTypes.TraceOptions, fn: (root: Span) => T): T {
     const rootSpan = new RootSpan(this, options);
+    // Add default attributes
+    const defaultAttributes = this.config && this.config.defaultAttributes;
+    if (defaultAttributes) {
+      Object.keys(defaultAttributes).forEach(key => {
```

Comment:
Could you switch this to a `for (const of` loop? That is better for the Closure compiler, which I'm hoping we will eventually use.